extra/docs_landing.html

```diff
@@ -48,335 +48,178 @@
-      </dd>
-    </dl>
-
-    <h3 id="_consume">Consume</h3>
-    <p>Use Kibana to query and visualize the data that&#8217;s stored in Elasticsearch. Or, use the Elasticsearch clients to access data in Elasticsearch directly from common programming languages.</p>
-    <dl>
-      <dt id="stack-components-kibana">
-        Kibana
-      </dt>
-      <dd>
-        Kibana is the tool to harness your Elasticsearch data and to manage the Elastic Stack. Use it to analyze and visualize the data that&#8217;s stored in Elasticsearch. Kibana is also the home for the Elastic Enterprise Search, Elastic Observability and Elastic Security solutions. <a href="en/kibana/current/introduction.html">Learn more about Kibana</a>.
-      </dd>
-      <dt id="stack-components-elasticsearch-clients">
-        Elasticsearch clients
-      </dt>
-      <dd>
-        The clients provide a convenient mechanism to manage API requests and responses to and from Elasticsearch from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. <a href="en/elasticsearch/client/index.html">Learn more about the Elasticsearch clients</a>.
-      </dd>
-    </dl>
-  </details>
-
-  <h2>Get started</h2>
-  <p>New to Elastic? Learn how you can make the most of your data with the Elastic Stack.
-    Get hands-on with a solution and quickly see data in action, or start from a blank page. </p>
-  <div class="row my-4">
+<div>
+  <p>
+    <a class="inline-block mr-3" href="https://www.elastic.co/guide/index.html#viewall">Browse all docs</a>
+    <a class="inline-block mr-3" href="https://www.elastic.co/guide/en/starting-with-the-elasticsearch-platform-and-its-solutions/current/new.html">Release docs</a>
+  </p>
+</div>
+<h2 style="font-size:20px;margin-top: 0px;margin-bottom: 0px !important;">Solutions</h2>
+  <div class="row">
     <div class="col-md-4 col-12 mb-2">
       <a class="no-text-decoration" href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index.html">
         <div class="card h-100">
-          <h4 class="mt-3">
+          <h3 class="mt-3">
             <span class="inline-block float-left icon mr-2" style="background-image: url('https://images.contentstack.io/v3/assets/bltefdd0b53724fa2ce/blt11200907c1c033aa/634d9da119d8652169cf9b2b/enterprise-search-logo-color-32px.png');"></span>
             Search
-          </h4>
+          </h3>
           <p>Build custom applications with your data using Elasticsearch.</p>
```

Comment:
If I understand correctly we're no longer linking to the "get started" content as currently with those first cards. So I'm wondering if we need to adjust this desc to introduce the whole solution? @chriscressman